FEXCore/Source/Interface/Core/HostFeatures.cpp

```diff
@@ -143,11 +150,13 @@ HostFeatures::HostFeatures() {
   SupportsSSE4A = true;
 #ifdef VIXL_SIMULATOR
   // Hardcode enable SVE with 256-bit wide registers.
-  SupportsSVE = true;
+  SupportsSVE128 = true;
+  SupportsSVE256 = true;
   SupportsAVX = true;
 #else
-  SupportsSVE = Features.Has(vixl::CPUFeatures::Feature::kSVE);
-  SupportsAVX = Features.Has(vixl::CPUFeatures::Feature::kSVE2) && vixl::aarch64::CPU::ReadSVEVectorLengthInBits() >= 256;
+  SupportsSVE128 = Features.Has(vixl::CPUFeatures::Feature::kSVE2);
+  SupportsSVE256 = Features.Has(vixl::CPUFeatures::Feature::kSVE2) && vixl::aarch64::CPU::ReadSVEVectorLengthInBits() >= 256;
+  SupportsAVX = SupportsSVE256;
```

Comment:
Pop this assignment out of the #if and then drop the `AVX = true` from the simulator